mkdocs.yml

```diff
@@ -11,52 +11,70 @@ markdown_extensions:
 pages:
 - Home: index.md
 - Basics:
-  - Developer Community: basics/community.md
-  - Prerequisite skills: basics/skills.md
-  - Requirements: requirements.md
-  - Planning your project: basics/planning.md
-  - Build: build.md
-  - Customize: customize.md
-  - Extend: extend.md
-  - Develop: develop.md
-  - Testing: testing.md
-  - Writing Documentation: documentation.md
-- Development Environment:
-  - Debugging: dev-tools/debugging.md
-- APIv3:
-  - API Intro: api/general.md
-  - API Usage: api/usage.md
-  - API Actions: api/actions.md
-  - API Parameters: api/params.md
+  - Developer Community: basics/community.md    # page-tree = DONE
+  - Prerequisite Skills: basics/skills.md       # page-tree = DONE
+  - Planning Your Project: basics/planning.md   # page-tree = DONE
+- Documentation:
```

Comment:
i think this meta section could be lower down (even at the end as much as i love docs)